src/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel='stylesheet' href='https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css'> 
+    <title>Weather App</title>
+</head>
+<body>
+    <header>

```

Comment:
Gr8 work 👏🏻 , please enclose the page within the div container. Maybe the screenshot u get an idea
##index.html 
![image](https://user-images.githubusercontent.com/99304440/182608260-4cd60c46-1125-4843-a231-4a004008e9ec.png)

##output screen
![image](https://user-images.githubusercontent.com/99304440/182608461-3856d0a2-af02-457c-845f-627a1adf86d8.png)